src/modules/menu/scss/topics-image.scss

```diff
@@ -34,16 +35,16 @@ $topics: (
 );
```

Comment:
you might want to reactivate prettier from here

Reply:
prettier-ignore-start and prettier-ignore-end are only supported on markdown. I could make it work only for this block but I'd have to wrap it into a block with brackets, which I'm not sure how it would interact with the code (unless I'm wrong, which is perfectly possible ^^).

and yes, I did format after rebasing.